pandas/tests/sparse/test_series.py

```diff
@@ -58,6 +60,9 @@ def _test_data2_zero():
 
 class TestSparseSeries(SharedWithSparse):
 
+    series_klass = SparseSeries
+    _assert_series_equal = staticmethod(tm.assert_sp_series_equal)
```

Comment:
This doesn't look as good, but avoids `*args, **kwargs` passing and retains docstring.